norse/torch/functional/receptive_field.py

```diff
@@ -158,9 +168,28 @@ def spatial_receptive_fields_with_derivatives(
     ]
 
 
-def temporal_scale_distribution(start: float, stop: float, steps: int):
-    return torch.exp(
-        torch.linspace(
-            torch.log(torch.as_tensor(start)), torch.log(torch.as_tensor(stop)), steps
-        )
-    )
+def temporal_scale_distribution(
+    n_scales: int,
+    min_scale: float = 1,
+    max_scale: Optional[float] = 1,
+    c: Optional[float] = 1.41421,
+):
+    r"""
+    Provides temporal scales according to [Lindeberg2016].
+    The scales will be logarithmic by default, but can be changed by providing other values for c.
+
+    .. math:
+        \tau_k = c^{2(k - K)} \tau_{max}
+        \mu_k = \sqrt(\tau_k - \tau_{k - 1})
+
+    Arguments:
+      n_scales (int): Number of scales to generate
+      min_scale (float): The minimum scale
+      max_scale (Optional[float]): The maximum scale, given the growth parameter c. Defaults to 1.
+      c (Optional[float]): The base from which to generate scale values. Should be a value between 1 to 2 exclusive. Defaults to sqrt(2).
+
+    .. [Lindeberg2016] Lindeberg 2016, Time-Causal and Time-Recursive Spatio-Temporal Receptive Fields, https://link.springer.com/article/10.1007/s10851-015-0613-9.
+    """
+    ks = torch.arange(min_scale, min_scale + n_scales + 1)
+    taus = c ** (2 * (ks - n_scales)) * max_scale
```

Comment:
Could someone provide both c and max_scale? I believe that in case where max_scale is provided, c should be calculated within the function with the formula $\ c = (\frac{max\textunderscore scale}{min\textunderscore scale})^{\frac{1}{2(n\textunderscore scales-1)}}$